LEAF_Request_Portal/admin/templates/mod_templates_reports.tpl

```diff
@@ -449,7 +446,6 @@
                 });
 
                 if (fileNames.indexOf(template) === -1) {
-                    console.log('Template file not found in directory');

```

Comment:
Since this just returns and does nothing would we still want some sort of notification?  The change above makes sense since there is a message that gets displayed to the user. The change below would be another example where we would want to notify the user. In this case it is just console messages, but I feel like we need to still let the user know something somehow.